webserver/src/test/java/io/kestra/webserver/controllers/BlueprintControllerTest.java

```diff
@@ -77,6 +76,7 @@ void blueprintFlow(WireMockRuntimeInfo wmRuntimeInfo) {
         wireMock.verifyThat(getRequestedFor(urlEqualTo("/v1/blueprints/id_1/flow")));
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
Well, I prefere to keep those warnings but ... OK-ish

Reply:
We've used them over 100 times in the application, so I don't see the need for spam to tell us that we're doing unchecked cast unless we plan to correct it in the short term.